### PR TITLE
DOCS-11170 Add ignored-by-default documentation.

### PR DESCRIPTION
**Pull request by giordanolucas**

### CODE REVIEW

File: modules/ROOT/pages/rest-sdk/configure-operations.adoc [hunk 1]
```diff
@@ -81,8 +80,7 @@ endpoints:
       ignored: false
 ----
 
-In this example, only the operations under the `/flights/{ID}` path will be generated in the connector,
-all other operations and triggers won't.
+In this example, only the operations and triggers under the `/flights/{ID}` path will be generated.
```
Comment (giordanolucas) [inline modules/ROOT/pages/rest-sdk/configure-operations.adoc]: @sathya0 I would revert this change, as triggers matching that path won't be generated, only the operations will.
Triggers can bu unignored in a different section.

File: modules/ROOT/pages/rest-sdk/configure-operations.adoc [hunk 2]
```diff
@@ -40,6 +40,49 @@ endpoints:
 In this case, all of the endpoint verbs will be ignored. No operations
 will be generated for this resource.
 
+== Ignore a Trigger
+
+If you want to ignore a trigger that was declared on a previous layer,
```
Comment (valkyrie69) [inline modules/ROOT/pages/rest-sdk/configure-operations.adoc]:
```suggestion
To ignore a trigger that was declared on a previous layer,
```

File: modules/ROOT/pages/rest-sdk/configure-operations.adoc [hunk 3]
```diff
@@ -40,6 +40,49 @@ endpoints:
...
+you can use the `ignored` property.
```
Comment (valkyrie69) [inline modules/ROOT/pages/rest-sdk/configure-operations.adoc]:
```suggestion
you can use the `ignored` property:
```

**Review by valkyrie69 — APPROVED** [review]
